web/js/diceware/Wordlist.js

```diff
@@ -0,0 +1,24 @@
+export { get };
+
+async function get(languageCode) {
+	try {
```

Comment:
I appreciate what you're doing here in terms of robustness of design. But under what circumstances do we envision an attempt to pass in a `languageCode` that isn't already known to be supported? Is that value coming from somewhere that the app itself doesn't control?